x-pack/plugins/infra/server/lib/alerting/metric_threshold/lib/metric_query.ts

```diff
@@ -74,10 +76,12 @@ export const getElasticsearchMetricQuery = (
           aggregatedIntervals: {
             date_range: {
               field: timefield,
-              ranges: Array.from(Array(Math.floor((to - from) / intervalAsMS)), (_, i) => ({
-                from: from + intervalAsMS * i,
-                to: from + intervalAsMS * (i + 1),
-              })),
+              ranges: [
+                {
+                  from: from - intervalAsMS,
```

Comment:
this should be `- deliveryDelay` and not `-intervalAsMS` right?

Reply:
actually, I think it should just be from. No need to subtract delivery delay